Resources/Prototypes/_DeltaV/Objectives/traitor.yml

```diff
@@ -0,0 +1,31 @@
+﻿# Kill
+- type: entity
+  parent: [BaseTraitorObjective, BaseTeachLessonObjective]
+  id: TeachLessonRandomPersonObjective
+  description: Kill them, and show everyone we mean business. They only need to die once.
+  components:
+  - type: Objective
+    difficulty: 1.75
+    unique: false
+  - type: TargetObjective
+    title: objective-condition-teach-person-title
+  - type: PickRandomPerson
+  - type: TeachLessonCondition
+# Base
+# teach lesson
+- type: entity
+  abstract: true
+  parent: BaseTargetObjective
+  id: BaseTeachLessonObjective
```

Comment:
Optional nitpick. Base is better located before the thing that inherits from it.

You can move it above `TeachLessonRandomPersonObjective`.